record-and-playback/core/scripts/sanity/sanity.rb

```
@@ -50,7 +50,7 @@
 
 def check_events_xml(raw_dir,meeting_id)
   filepath = "#{raw_dir}/#{meeting_id}/events.xml"
-  raise Exception,  "Events file doesn't exists." if not File.exists?(filepath)
+  raise Exception, "Events file doesn't exists." if not File.exist?(filepath)
```

RuboCop (1.88.0)

[convention] Add empty line after guard clause.

(Layout/EmptyLineAfterGuardClause)